scoring_engine/web/views/api/overview.py

```diff
@@ -42,31 +37,28 @@ def overview_get_round_data():
 
 
 @mod.route('/api/overview/data')
-@cache.memoize()
 def overview_data():
     team_data = OrderedDict()
-    teams = session.query(Team).filter(Team.color == 'Blue').order_by(Team.id).all()
-    random.shuffle(teams)
-    for team in teams:
+    blue_teams = Team.get_all_blue_teams_cached()
+    random.shuffle(blue_teams)
+    for team in blue_teams:
         service_data = {}
-        for service in team.services:
-            service_data[service.name] = {
-                'passing': service.last_check_result(),
-                'host': service.host,
-                'port': service.port,
+        for service in team['services']:
```

Comment:
Instead of having to access things by a dict, I wonder if we can do something like a DotDict, where we can access keys by doing team.services.